code/modules/projectiles/projectile/special/neurotoxin.dm

```diff
@@ -12,3 +12,21 @@
 		var/mob/living/carbon/H = target
 		H.reagents.add_reagent(/datum/reagent/toxin/staminatoxin/neurotoxin_alien, 10 * H.get_permeability())
 	return ..()
+
+/obj/item/projectile/bullet/acid
+	name = "acid spit"
+	icon_state = "neurotoxin"
+	damage = 2
+	damage_type = BURN
+	flag = BIO
+	range = 7
+	speed = 1.8 // spit is not very fast
+
+obj/item/projectile/bullet/acid/on_hit(atom/target, blocked = FALSE)
+	if(isalien(target)) // shouldn't work on xenos
+		nodamage = TRUE
+	else if(isobj(target) || isclosedturf(target))
+		target.acid_act(acidpwr = 50, acid_volume = 25) // does good damage to objects and structures
+	else if(iscarbon(target))
+		target.acid_act(acidpwr = 18, acid_volume = 25) // balanced
```

Comment:
```suggestion
	else if(iscarbon(target))
		target.acid_act(acidpwr = 18, acid_volume = 25) // balanced
	else if(!isopenturf(target)
		target.acid_act(acidpwr = 50, acid_volume = 25) // does good damage to objects and structures
```

Reply:
no reason to restrict it so arbitrarily, all it needs to do is not be able to melt through the floor